pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfigProperties.java

```diff
@@ -42,7 +42,11 @@
   public static final String SCHEMA_URI = "schemaURI";
   public static final String SEQUENCE_ID = "sequenceId";
   public static final String SEGMENT_NAME_GENERATOR_TYPE = "segmentNameGenerator.type";
-  public static final String SEGMENT_NAME_GENERATOR_CONFIGS = "segmentNameGenerator.configs";
+  public static final String SEGMENT_NAME_GENERATOR_PROP_PREFIX = "segmentNameGenerator.configs";
+  public static final String SEGMENT_NAME = "segment.name";
+  public static final String SEGMENT_NAME_PREFIX = "segment.name.prefix";
+  public static final String SEGMENT_NAME_POSTFIX = "segment.name.postfix";
```

Comment:
nit: suffix

Reply:
I tried to keep the property names same as the ones in the existing yml batch job spec. It uses postfix there